namer/namer.cc

```diff
@@ -1334,8 +1339,9 @@ class SymbolDefiner {
             // otherwise, we're looking at a type member defined in this class in the same file, which means all we
             // need to do is find out whether there was a redefinition the first time, and in that case display the
             // same error
-            auto oldSym = ctx.state.findRenamedSymbol(onSymbol, existingTypeMember);
-            if (oldSym.exists()) {
+            auto name = existingTypeMember.data(ctx)->name;
+            if (isMangleRenameUniqueName(ctx, name)) {
+                auto oldSym = ctx.state.lookupSymbol(onSymbol, typeMember.name);
```

Comment:
```suggestion
                auto oldSym = ctx.state.lookupSymbol(onSymbol, name);
```

Reply:
I don't think this is equivalent? I think in this case, we're guaranteed that `name` is a MangleRename unique name, which means that if we look up the symbol with that name, we're just going to find the mangled symbol again. What we want with this lookupSymbol call is to find the name that's **not** mangled, which we can do by using `typeMember.name`, because `typeMember.name` came from the FoundDefinition, and the found definitions never have mangled names (because users can't write mangled names).